packages/cli/src/databases/entities/test-run.ee.ts

```diff
@@ -35,4 +35,17 @@ export class TestRun extends WithTimestampsAndStringId {
 
 	@Column(jsonColumnType, { nullable: true })
 	metrics: AggregatedTestRunMetrics;
+
+	// Total number of the test cases, matching the filter condition of the test definition (specified annotationTag)
```

Comment:
Could we use JSDoc for these comments, so editors can show it when hovering the property